website/docs/r/batch_pool.html.markdown

```diff
@@ -120,6 +149,8 @@ The following arguments are supported:
 
 * `storage_image_reference` - (Required) A `storage_image_reference` for the virtual machines that will compose the Batch pool.
```

Comment:
I don't see any docs for the block here, could we add that in?

Reply:
added.